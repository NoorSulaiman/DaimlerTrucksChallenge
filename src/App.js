import "./App.css";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import SearchPartsPage from "./pages/SearchParts/SearchPartsPage";
import DeletePartsPage from "./pages/DeleteParts/DeletePartsPage";
function App() {
  return (
    <div className="App">
      <Router>
        <Switch>
          <Route path="/" exact component={SearchPartsPage} />
          <Route path="/delete" component={DeletePartsPage} />
        </Switch>
      </Router>
    </div>
  );
}

export default App;
