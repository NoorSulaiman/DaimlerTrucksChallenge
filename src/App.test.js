import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("App", () => {
  it("should render App correctly with given props", () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
