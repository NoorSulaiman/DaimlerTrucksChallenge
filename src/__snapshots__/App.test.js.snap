// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render App correctly with given props 1`] = `ShallowWrapper {}`;
